doc/design/plugin-conf.md

```diff
@@ -22,7 +22,14 @@ scenario on the fly.
 The following YAML format will be introduced for dynamic plugin configuration:
 
 ```yaml
-actions: "list_of_action_in_order"
+actions:
+- name: action_1
+  options:
```

Comment:
remove `options`, refer to plugin arguments.

Reply:
type PluginOption struct {
	// The name of Plugin
	Name string `yaml:"name"`
        .......
	// Arguments defines the different arguments that can be given to different plugins
	Arguments map[string]string `yaml:"arguments"`
}

if there's plugin arguments, it will be:

- name: xxx
   arguments: 

I can rename option to arguments.

Reply:
renamed to "arguments"